commands/create.go

```diff
@@ -221,7 +223,7 @@ func createCmd(dockerCli command.Cli) *cobra.Command {
 	flags := cmd.Flags()
 
 	flags.StringVar(&options.name, "name", "", "Builder instance name")
-	flags.StringVar(&options.driver, "driver", "", fmt.Sprintf("Driver to use (available: %v)", drivers))
+	flags.StringVar(&options.driver, "driver", "", fmt.Sprintf("Driver to use (available: `%v`)", drivers))
```

Comment:
Not a stopper for this PR, but wondering if we should reformat the `drivers` to the same (comma-separated) format we use for some other options (e.g., `--progress` shows: `(auto, plain, tty)`).

Reply:
Sure will do that